src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -365,6 +368,54 @@ private static boolean compareFiles(
     return didCompare;
   }
 
+  /**
+   * Rather than storing both the extended summary and non-extended summary files in git, we can
+   * directly check that the non-extended file is precisely what we expect: everything in the
+   * extended file except for the inactive ballot breakdown.
+   */
+  private static void compareExtendedSummaryToSummary(
+          ContestConfig config, String timestampString, String sequentialId) {
+    String summaryPath = ResultsWriter.getOutputFilePath(
+            config.getOutputDirectory(), "summary", timestampString, sequentialId)
+            + ".csv";
+    String extendedPath = ResultsWriter.getOutputFilePath(
+            config.getOutputDirectory(), "extended_summary", timestampString, sequentialId)
+            + ".csv";
+
+    try (BufferedReader brSummary = new BufferedReader(new FileReader(summaryPath, UTF_8));
+         BufferedReader brExtended = new BufferedReader(new FileReader(extendedPath, UTF_8))) {
+      while (true) {
+        String lineExtended = brExtended.readLine();
+        // If the extended file has reached its end, then the non-extended file must have too
+        if (lineExtended == null) {
+          assertNull(brSummary.readLine(), "Extended file is missing a line");
+          return;
+        }
+
+        // If the extended file should be excluded, continue without moving the file pointer
+        // in the non-extended file. For now, there's only one type of row excluded, and they
+        // happen to all start with "Inactive Ballots by"
+        if (lineExtended.startsWith("Inactive Ballots by")) {
+          continue;
+        }
+
+        // This line should be equal in both files. Ensure the line exists and they're equal
+        // in both files.
+        String lineSummary = brSummary.readLine();
+        assertNotNull(lineSummary, "Summary file is missing a line");
+        if (!lineSummary.equals(lineExtended)) {
+          fail();
```

Comment:
So I made a change to force a mismatch between the summary and extended_summary file text. It caused this line to fail as expected, but it was just a generic stack trace that was hard to hunt down. Is it possible to change this to include a more human readable failure message like "Line X mismatch between summary and extended_summary"?

Reply:
done!